surveys/models.py

```diff
@@ -81,6 +81,9 @@ class Survey(models.Model):
     def __str__(self):
         return self.name
 
+    def get_api_url(self):
+        return drf_reverse('survey-form', kwargs={'pk': self.pk})
```

Comment:
We'll need it for the API description tests coming up soonish.  I figured I'd throw it in.